modules/01 JavaScript Programming/Assignments/19/answerJ19.txt

```diff
@@ -0,0 +1,125 @@
+
+var errorsObjects={
+    valid:true,
```

Comment:
Semantically placing "valid" inside "errorObject" means that "errorObject" is valid. But it's not your purpose. This "valid" doesn't indicate whether "errorObject" is valid or not.